src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -129,6 +136,44 @@ public function panelAction(Request $request, $token)
     }
 
     /**
+     * Replays an action.
+     *
+     * @param Request $request The current HTTP Request
+     * @param string  $token   The profiler token
+     *
+     * @return RedirectResponse A RedirectResponse instance
+     */
+    public function replayAction(Request $request, $token)
+    {
+        $profile = $this->profiler->loadProfile($token);
+
+        $replayer = new Client($kernel = clone $this->kernel);
+        $kernel->boot();
+        $replayer->enableProfiler();
+
+        /** @var RequestDataCollector $requestCollector */
+        $requestCollector = $profile->getCollector('request');
+        $parameters = array();
+        /** @var Data $data */
+        foreach (array_merge($requestCollector->getRequestQuery()->all(), $requestCollector->getRequestRequest()->all()) as $k => $data) {
+            $parameters[$k] = $data->getValue(true);
```

Comment:
merging the GET and POST parameters together does not make any sense. You are not replyaying the original request, but a new one built with weird content.